configs/faster_rcnn/metafile.yml

```diff
@@ -276,7 +276,8 @@ Models:
         Dataset: COCO
         Metrics:
           box AP: 37.9
-    Weights: https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_iou_1x_coco-fdd207f3.pth
+    # re-release
```

Comment:
Seems not necessary?

Reply:
The checkpoint is same as the original one, but the link is updated.